src/control/lib/daos/pool_property.go

```diff
@@ -27,6 +28,42 @@ func numericMarshaler(v *PoolPropertyValue) ([]byte, error) {
 	return json.Marshal(n)
 }
 
+const (
+	defDurationUnit = time.Second
+)
+
+// parseDuration parses a duration string into a time.Duration.
+func parseDuration(s string) (time.Duration, error) {
+	if s == "" {
+		return 0, nil
+	}
+
+	// Append the duration unit if it is missing.
+	if s[len(s)-1] >= '0' && s[len(s)-1] <= '9' {
+		dduStr := defDurationUnit.String()
+		s += dduStr[len(dduStr)-1:]
```

Comment:
As I think about it, this isn't really durable if we changed the default unit to anything containing multiple letters (e.g. ms). Might be better to just define a string "s" as the default instead of deriving it from `time.Second`.

Reply:
Yeah... I was just tacking on a "s" but tried to get fancy. I'll go back to the original approach.